### PR TITLE
Return sacrifice icon to item search

### PR DESCRIPTION
**Pull request by nwjgit**
Returns the :incinerator: emoji back to item search if the user has sacrificed that item.

fixes: https://github.com/oldschoolgg/oldschoolbot/issues/5344

- [X] I have tested all my changes thoroughly.
Discord_Yxxk1vHRER

### CODE REVIEW

File: src/lib/events.ts [hunk 1]
```diff
@@ -197,6 +198,7 @@ const mentionCommands: MentionCommand[] = [
 
 					if (user.cl.has(item.id)) icons.push(Emoji.CollectionLog);
 					if (user.bank.has(item.id)) icons.push(Emoji.Bank);
+					if ((sacrificedBank as ItemBank)[item.id]) icons.push(Emoji.Incinerator);
```
Comment (themrrobert) [inline src/lib/events.ts:201]: Please fix the warning associated with your change. 

Perhaps:
```js
if ((sacrificedBank as ItemBank)[item.id] ?? 0 > 0) icons.push(Emoji.Incinerator);
```

**Review by themrrobert — CHANGES_REQUESTED** [review]

File: src/lib/events.ts (hunk 1 above)
Comment (nwjgit) [reply src/lib/events.ts]: Threw the same warning, I believe my new commit solves the issue.

**Comment by nwjgit** [comment]
Fixed warning and tested.

- [X] I have tested all my changes thoroughly.
Discord_zeb72eYwyn

**Comment by themrrobert** [comment]
> Fixed warning and tested.
> 
> * [x]  I have tested all my changes thoroughly.
>   Discord_zeb72eYwyn

Could you try this instead?

```ts
if (((sacrificedBank as ItemBank)[item.id] ?? 0) > 0) icons.push(Emoji.Incinerator);
```

I believe that should fix the issue with my original line. Apparently null coalesce has very low precedence.

Note: You will have to `git pull` the changes since we merged the master branch into your branch.

**Review by themrrobert — APPROVED** [review]

**Pull request merged**